src/App/DocumentObject.cpp

```diff
@@ -838,8 +838,7 @@ DocumentObject::onProposedLabelChange(std::string& newLabel)
     }
     static ParameterGrp::handle _hPGrp;
     if (!_hPGrp) {
-        _hPGrp = GetApplication().GetUserParameter().GetGroup("BaseApp");
-        _hPGrp = _hPGrp->GetGroup("Preferences")->GetGroup("Document");
+        _hPGrp = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/Document")

```

Comment:
Missing semicolon.

Reply:
That is why I don't like the "Suggest change" feature...

Reply:
Oh, hey, I didn't know that was a thing.  Let me get a quick suggestion in to fix it!